src/commonTest/kotlin/ParserTest.kt

```diff
@@ -6,10 +6,10 @@ import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
-open class ParserTest {
+class ParserTest {
 
     @Test
-    fun `Command without Parameters and no args should return empty maps`() {
+    fun `Command_without_Parameters_and_no_args_should_return_empty_maps`() {
```

Comment:
I know the backticks are a dirty Kotlin Compiler hack (for another use case), but I liked them. Unfortunately, they aren't allowed for JS projects (`Name contains illegal chars that can't appear in JavaScript identifier`).

Please remove the redundant backticks and we do it the old fashion way. Yes, underscore is some strange naming, too. But I like it for tests :)

Reply:
👍 Got it.